nss/src/cache/mod.rs

```diff
@@ -26,6 +26,10 @@ pub const PASSWD_PERMS: u32 = 0o644;
 const SHADOW_DB: &str = "shadow.db"; // Ownership: root:shadow
 pub const SHADOW_PERMS: u32 = 0o640;
 
+const DB_CON_PREFIX: &str = "file:"; // Specify "file" as access mode, so that we can use connection options.
+const DB_CON_OPT_RW: &str = "?journal_mode=wal"; // Use Write Ahead Log journaling mode, so that we can operate paralell with the PAM module.
+const DB_CON_OPT_RO: &str = "?immutable=1"; // When using immutable=1, we can still read the passwd_db, even if there is an db lock.
```

Comment:
Minor nitpick here: I would refactor these names to use `CONN` instead of `CON`, as it gives a better idea of what this value refers to.

Reply:
Thanks! good catch, its chaneged in the latest commit.